src/adservice/Dockerfile

```diff
@@ -22,5 +22,8 @@ RUN GRPC_HEALTH_PROBE_VERSION=v0.3.1 && \
 WORKDIR /app
 COPY --from=builder /app .
 
+# Allow bottlenecks via copyright certification
+ENV ENABLE_COPYRIGHT_CERTIFICATION=true
+
 EXPOSE 9555
-ENTRYPOINT ["/app/build/install/hipstershop/bin/AdService"]
```

Comment:
Curious how the path appeared to be changed?

Reply:
Yeah, so the custom gradle (java build tool) magic that was there before was setting it up in a custom path. It's using a more standard gradle default now.

Reply:
Thanks for clarification!